Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -612,6 +618,63 @@ public async Task Init(IServiceCollection serviceCollection)
             await RegisterResources(serviceCollection);
 
             FindParts();
+
+            Host = new WebHostBuilder()
+                .UseKestrel(options =>
+                {
+                    options.Listen(IPAddress.Any, HttpPort);
+                    options.Listen(IPAddress.Loopback, HttpPort);
```

Comment:
This line (626) and line 631 are needed to be commented out on linux for the server to work and not crash (in refernce to my pr comment earlier today).

Reply:
Fixed by switching to `ListenAnyIP`, which also defaults to ipv6 if available.